common/beerocks/bwl/brcm/ap_wlan_hal_brcm.h

```diff
@@ -9,8 +9,8 @@
 #ifndef _BWL_AP_WLAN_HAL_BRCM_H_
 #define _BWL_AP_WLAN_HAL_BRCM_H_
 
-#include "../common/ap_wlan_hal.h"
 #include "base_wlan_hal_brcm.h"
+#include <ap_wlan_hal.h>
```

Comment:
I think that relative paths for files in the same module is better practice.
Can prevent issues when files from different modules have the same name.

This comment is relevant for all BPL/BWL files.

Reply:
The idea is to be able to easilly move the BWL/BPL to the framework so we don't maintain 2 different common libraries. I also agree with you that using only the file name is problematic - so how about a compromise, adding a `bwl` directory prefix? for example, `#include <bwl/base_wlan_hal.h>`

Reply:
If you agree with this approach, I will add this change in a separate commit right after this one.

Reply:
You can keep the relative paths and still move the modules between the projects.
Relative paths are used only for including files from the same logical group of files (e.g. inside the BWL module)

Reply:
So you think this whole commit should be removed?

Reply:
On second thought we decided to keep Arnout's change but to use relative paths to `<bwl/...>` `<bpl/,,,>` `<btl/,,` etc. So I will change this commit by Arnout accordingly.